composer.json

```diff
@@ -3,12 +3,12 @@
     "description": "PHP CS Fixer config for Paysera conventions",
     "type": "library",
     "require": {
-        "php": ">=7.4",
+        "php": "^7.4 || ^8.1",
         "doctrine/inflector": "^1.0 || ^2.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.0 || ^7.0 || ^8.0 || ^9.0",
-        "friendsofphp/php-cs-fixer": "3.60.0",
+        "phpunit/phpunit": "^9.3.0",
```

Comment:
will it still work on PHP 7.4 with these phpunit requirements?

Reply:
Requirements for 9.3.0 are php: ^7.3 || ^8.0 and php: >=7.3 for 9.6.21 (last 9.x version for now). I also test it locally with PHP 7.4 and it works. So I believe it should be ok.